webpack.common.js

```diff
@@ -91,11 +90,20 @@ module.exports = {
             },
             {
                 test: /\.js$/,
-                exclude: /node_modules[\\/](?!date-fns|epubjs)/,
+                exclude: /node_modules[\\/](?!@uupaa[\\/]dynamic-import-polyfill|date-fns|epubjs|flv.js|libarchive.js)/,
+                use: [{
+                    loader: 'babel-loader'
+                }]
+            },
+            /* modules that Babel breaks when transforming to ESM */
+            {
+                test: /node_modules[\\/](pdfjs-dist|xmldom)[\\/].*\.js$/,
                 use: [{
                     loader: 'babel-loader',
                     options: {
-                        presets: packageConfig.babel.presets
+                        plugins: [
+                            '@babel/transform-modules-umd'
```

Comment:
Does this not need included as a dev dependency?

Reply:
:+1:  It is safer to add